aboutme.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Assets/aboutme.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <!-- This is a header section with nav bar to navigate through other sections of the page -->
+    <header>
+      <h1>About Me</h1>
+    </header>
+    <nav>
```

Comment:
not sure if perhaps the wrong branch has been published but unfotrunately the nav is not visible at the deployed link - might be worth a quick look